app/components/views/AccountsPage/Privacy/Page.js

```diff
@@ -0,0 +1,88 @@
+import { FormattedMessage as T } from "react-intl";
+import { TextInput, NumericInput } from "inputs";
+import { Subtitle } from "shared";
+import { PassphraseModalSwitch, AutoBuyerSwitch } from "buttons";
+import cx from "classnames";
+import "style/Privacy.less";
+
+const PrivacyPage = ({
+  mixedAccountName, setMixedAccountBranch, mixedAccountBranch, changeAccountName,
+  canStartMixer, error, accountMixerRunning, stopAccountMixer, onStartMixerAttempt,
+  csppServer, csppPort
+}) => (
+  <>
+    <Subtitle title={<T id="privacy.subtitle" m="Privacy"/>} />
+
+    <div className="privacy-page-wrapper is-column">
+      <div className={ cx("is-row", "privacy-row") }>
+        <div className={ cx("is-row", "privacy-item", error["mixedAccount"] && "error") }>
+          <div className=""><T id="privacy.mixing.account" m="Mixing Account" />:</div>
+          <TextInput required disabled value={mixedAccountName} />
+        </div>
+
+        <div className={cx("privacy-item", error["mixedAccountBranch"] && "error")}>
+          <div className="is-row">
+            <div className=""><T id="privacy.mixing.account.branch" m="Account Branch" />:</div>
+            <NumericInput
+              value={mixedAccountBranch} onChange={ e => setMixedAccountBranch(e.target.value)}
```

Comment:
I think this should be disabled as well, following the convention we're setting of creating the mix config for the user.

Otherwise, at the very least, this needs to be protected against changing from any value besides {0,1}.